[CI:DOCS] Update docs/tutorials/rootless_tutorial.md:

ebb-earl-co opened the pull request:
* Change references of 'master' to 'main' in URLs e.g. https://github.com/containers/podman/blob/main/install.md
* Wrap names of files or programs by '`' e.g. `dnf`, `containers.conf`, `/etc/subuid`, etc.
* Change sentence with ambiguous subject to 'Root privileges are required to add or update entries within these files'
* Link to kernel.org documentation for the `getpwent` command
* Change sentence: 'Note that the values for each user must be unique ~and without any overlap~'
* Make references to the Podman project upper-case instead of lower-case
* Reorder sentence 'Update the `/etc/subuid` and `/etc/subgid` with fields for each user' to emphasize 'For each user'

Signed-off-by: Colin Eberl Coe <ebb-earl-co@pm.me>

<!--
Thanks for sending a pull request!

Please make sure you've read our contributing guidelines and how to submit a pull request (https://github.com/containers/podman/blob/master/CONTRIBUTING.md#submitting-pull-requests).

In case you're only changing docs, make sure to prefix the pull-request title with "[CI:DOCS]".  That will prevent functional tests from running and save time and energy.
-->

ebb-earl-co commented:
/assign @saschagrunert

ebb-earl-co commented:
CI failed with the following: 
```- FAIL - commit subject exceeds 90 characters```
should the commit message for the PR be drastically shortened, with the list of changes added as a comment here? If so, to fit all the doc changes into 90 characters seems like a tall task if the message is still to be meaningful...

Luap99 commented:
> CI failed with the following:
> `- FAIL - commit subject exceeds 90 characters`
> should the commit message for the PR be drastically shortened, with the list of changes added as a comment here? If so, to fit all the doc changes into 90 characters seems like a tall task if the message is still to be meaningful...

You are likely missing an empty newline after your first line in the git commit message. You can change the message with `git commit --amend` and force push. I think this should do it.

ebb-earl-co commented:
@Luap99 thank you for pointing that out! Because I made so many changes, I had them all in a file and so did `git commit -F` for the first time; that will have done it. Fingers crossed that that's all that needed changed!

rhatdan commented:
You need to rebase and squash your commits.

git rebase -i origin

ebb-earl-co commented:
@rhatdan thank you for the suggestion; I rebased and squashed my commits into one, nicely-formatted message, and it seems that that has gotten me over the previous hurdle.
Now, I don't know why the CI failed: the logs mention an `AssertionError` coming out of Python because of a UID/GID mismatch. However, the link to the test [annotations](https://storage.googleapis.com/cirrus-ci-6707778565701632-fcae48/artifacts/containers/podman/6104795787296768/html/apiv2-podman-fedora-34-root-host.log.html) shows 823 test passed out of 823 tests, so I'm not sure what gives.

Luap99 reviewed (approved):
LGTM

rhatdan commented:
/lgtm

rhatdan commented:
Thanks @ebb-earl-co

ebb-earl-co commented:
@rhatdan my pleasure! Glad to have contributed
